src/main/webapp/trips/tripScript.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Triggered upon DOM load.
+$(document).ready(() => {
+  // Redirect to homepage if user is not signed in.
+  isSignedIn().then((signInStatus) => {
+    if (!signInStatus) {
+      window.location.replace('/');
+    } else {
+      setContentWidth('800px');
```

Comment:
another comment that you don't really need to act on right now, but it's good to think about. Be wary of hard-coding pixel sizes, since they'll look differently on different sized windows.

But, again, just food for thought

Reply:
Agreed, I'll keep that in mind -- maybe I can base this on screen size (some percentage of the width) in the future.